src/granite/table.cr

```diff
@@ -51,12 +51,12 @@ module Granite::Tables
   end
 
   macro table(name)
-    @[Granite::Table(name: {{(name.is_a?(StringLiteral) ? name : name.stringify) || nil}})]
+    @[Granite::Table(name: {{(name.is_a?(StringLiteral) ? name : name.id.stringify) || nil}})]
```

Comment:
@Blacksmoke16 by adding the `.id`, this adds support for symbols so you can do `table :posts`.